libs/wire-api/src/Wire/API/User/IdentityProvider.hs

```diff
@@ -169,6 +170,10 @@ instance ToJSON IdPMetadataInfo where
   toJSON (IdPMetadataValue _ x) =
     object ["value" .= SAML.encode x]
 
+-- | (Returning 'Nothing' would be an internal error.)
```

Comment:
Could we encode this better in the types?

Reply:
c22b9602421ffe7f266982faabb60419f06ca90b